### PR TITLE
MF-381: Rewrite patient search results UI based on carbon designs

### PR DESCRIPTION
**Pull request by denniskigen**
[Ticket](https://issues.openmrs.org/browse/MF-381)

This PR updates the patient search results UI to match the latest [designs](https://app.zeplin.io/project/5f7223cfda10f94d67cec6d0/screen/5fa44069a6175631e853c3ba). As part of this effort, it also adds rudimentary support for SASS and image files (for use as patient avatars).

Screenshot 2020-12-03 at 12 07 10

empty-search-results

TODO:

- Add pagination controls using the [PaginationNav](https://react.carbondesignsystem.com/?path=/story/paginationnav--pagination-nav) component.

### CODE REVIEW

**Review by FlorianRappl — APPROVED** [review]
LGTM - :beers:!

**Comment by jonathandick** [comment]
@denniskigen nice work. Seems like has bee approved. Did you want to move from draft to PR?

**Comment by denniskigen** [comment]
What remains is adding the PaginationNav. I can finish that tomorrow and wrap this up entirely.

File: src/types.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  person: {
+    age: number;
+    birthdate: string;
+    display: string;
+    gender: string;
+  };
+  display: string;
+}
+
+export type Identifier = {
```
Comment (FlorianRappl) [inline src/types.ts:14]: Its strange that the upper one (`SearchedPatient`) is an interface, while this was defined as a `type`. Maybe make both `interface`s as this is what they are.

**Review by FlorianRappl — COMMENTED** [review]
LGTM - thanks! :beers:

File: src/types.ts (hunk 1 above)
Comment (denniskigen) [reply src/types.ts]: Fixed

**Pull request merged**